lib/wallaby/browser.ex

```diff
@@ -392,12 +392,37 @@ defmodule Wallaby.Browser do
   @doc """
   Sets the value of an element.
```

Comment:
Since we're explicitly looking for `:selected` and `:unselected` we'll want to call that out in the documentation.